src/components/navbar/navbar.css

```diff
@@ -0,0 +1,84 @@
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1.5rem;
+  position: sticky;
+  z-index: 100;
+  top: 0;
+  background-color: white;
+}
+
+.nav-left {
+  display: flex;
+  align-items: center;
+}
+.navbar .link {
+  text-decoration: none;
+}
+.nav-logo {
+  font-size: 1.7rem;
+  color: var(--primary-voilet);
+  letter-spacing: 1px;
+}
+
+.nav-input {
+  display: flex;
+  align-items: center;
+  width: 40%;
+  border: 2px solid lightgray;
```

Comment:
Suggestion - Use color variables instead of using direct colors, it will help change the colors in one place easily